src/reference/modules/installation/pages/upgrade.adoc

```diff
@@ -1,5 +1,49 @@
 = Upgrade notes
 
+== Upgrade from a previous Rudder 5.0
+
+Migration from any 5.0 minor version is supported.
```

Comment:
this is misleading, we can also upgrade from 4.1 and 4.3

Reply:
i know it's written below, but usually we stop at the first relevant information